src/openzaak/notifications/default.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2019 - 2022 Dimpact
+from openzaak.components.zaken.models import ZaakObject
+
+
+def handle(message: dict) -> None:
+    resource = message.get("resource")
+    actie = message.get("actie")
+    resource_url = message.get("resourceUrl")
+    kenmerken = message.get("kenmerken", {})
+    objecttype_omschrijving = kenmerken.get("objecttypeOmschrijving")
+    zaken_ids = message.get("zaken", [])
```

Comment:
I *think* the `zaken` field will contain URLs, so you can't directly do filtering by `zaak_id` (because those are the internal ids of the zaken.

I think what you could do is instantiate a `BaseLoader` and check for each URL if it is local (if it isn't, you can ignore it) and if it is, load the object with https://github.com/maykinmedia/django-loose-fk/blob/main/django_loose_fk/loaders.py#L69

Then you can use those instances in the code below